lib/src/base-styles/_nx-lists.scss

```diff
@@ -20,101 +19,94 @@
   }
 }
 
-.nx-list__title {
-  color: $nx-text-color;
-  margin: 0;
-}
-
 .nx-list__item {
-  @include semi-bold();
-  @include nx-truncate-ellipsis();
+  @include container-horizontal;
   @include container-vertical;
 
-  border-top: 1px solid transparent;
-  border-bottom: $nx-border-row;
+  border-top: 1px solid #e9e9e9;
   box-sizing: border-box;
-  color: $nx-text-color;
+  display: grid;
+  grid-template-columns: 1fr auto;
+  grid-template-rows: auto auto;
+  grid-template-areas: 
+    "list-text actions"
+    "list-sub-text actions";
   margin: 0;
-  padding: $nx-spacing-sm $nx-spacing-xs;
+  padding: $nx-spacing-md;
 
   // we use first of type because we only want to modify the first list item,
   // not alerts, errors, or empty states
   &:first-of-type {
-    border-top: $nx-border;
+    border-top: 1px solid #d1d6f0;
   }
+}
 
-  &.nx-error {
-    border: $nx-border-error;
-    color: $nx-text-color-error;
-  }
+.nx-list__text {
+  @include semi-bold();
 
-  > .fa, > .svg-inline--fa {
-    margin-right: $nx-spacing-md;
-  }
+  grid-area: list-text;
 }
 
-.nx-list--bulleted {
-  .nx-list__item {
-    @include regular();
+.nx-list__subtext {
+  @include regular();
 
-    border: none;
-    list-style: disc outside;
-    margin: $nx-spacing-sm 10px $nx-spacing-sm 36px;
-    overflow: visible;
-    padding: 0;
-    white-space: normal;
-  }
+  display: block;
+  grid-area: list-sub-text;
+  // top margin is measured down from the top of the objects grid row
+  margin: $nx-spacing-md 0 0 0;
 
-  .nx-list__item {
-    list-style-type: disc;
+  &.nx-truncate-ellipsis {
+    white-space: nowrap;
+  }
+}
 
-    .nx-list__item {
-      list-style-type: circle;
+.nx-list__item--empty {
+  display: list-item;
+  text-align: center;
 
-      .nx-list__item {
-        list-style-type: square;
-      }
-    }
+  .nx-list__text {
+    @include regular();
   }
 }
 
-.nx-list__item--empty {
-  @include regular();
+.nx-list__item--error {
+  display: list-item;
 
-  border-top: $nx-border; // assume it's always the first item
-  border-bottom: $nx-border-row;
-  color: $nx-text-color-light;
-  text-align: center;
+  .nx-alert {
+    margin-top: 0;
+  }
 }
 
-.nx-list__item--disabled {
+.nx-list__item.disabled {
   color: $nx-text-color-disabled;
   cursor: default;
 }
 
-.nx-list__item--expanding {
-  height: auto;
-}
+.nx-list--bulleted {
+  margin-left: $nx-spacing-xs;
 
-.nx-list__item--with-modifier-icon {
-  line-height: 30px;
-}
+  .nx-list__item {
+    border: none;
+    display: list-item;
+    list-style: disc outside;
+    margin: $nx-spacing-xs $nx-spacing-md;
```

Comment:
> I think the left margin here needs to be 18px to get the bullets in the right place. Additionally I think a -6px left margin on `.nx-list__text` is needed to get the spacing between the bullet and the text right.

Ok: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/dfefa153f3918369ad4542d1b0d163dd88628852

Reply:
> Ok: dfefa15

It looks like to exactly match what's in zeplin a small negative margin would also be necessary on the nested `<ul>` elements.  Right now the nested list is 5px indented from the text of its parent item, in zeplin it's 2px.  It may be worth clarifying with @dturner32 , as these are all very custom numbers.

Reply:
> > Ok: [dfefa15](https://github.com/sonatype/sonatype-react-shared-components/commit/dfefa153f3918369ad4542d1b0d163dd88628852)
> 
> It looks like to exactly match what's in zeplin a small negative margin would also be necessary on the nested `<ul>` elements. Right now the nested list is 5px indented from the text of its parent item, in zeplin it's 2px. It may be worth clarifying with @dturner32 , as these are all very custom numbers.

I think I have it dead on here (I measured screenshots): https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/af234cc3d1786625c56e3b13b2069f892488b6c2